workers/c_client_vtable/client.c

```diff
@@ -173,8 +173,7 @@ int main(int argc, char** argv) {
   command_request.user_handle = CreateCommandObject(1, &take_control_request);
```

Comment:
Don't forget to set `command_request.command_index = 1;` here.

Reply:
Good spot, fixed.